modules/tester/manifests/init.pp

```diff
@@ -17,7 +17,7 @@
 			ensure => directory,
 		}
 
-		if ( $config[php] < 5.6 ) {
+		if ( $config[php] < '5.6' ) {
```

Comment:
I'm 99% certain we actually need to use `versioncmp` as per the [php.pp code](https://github.com/Chassis/Chassis/blob/fcb4e646ba821b78edba92ad76ea6e0585f211c2/puppet/modules/chassis/manifests/php.pp#L35)?